base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -675,6 +697,109 @@ impl ConsensusConstants {
     }
 }
 
+#[derive(PartialEq)]
+enum CheckDifficultyRatio {
+    Yes,
+    No,
+}
+
+#[derive(PartialEq)]
+enum CheckLwmaRatios {
+    Yes,
+    No,
+}
+
+// Assert the hybrid POW constants.
+// Note: The math and constants in this function should not be changed without ample consideration that should include
+//       discussion with the Tari community, modelling and system level tests.
+// For SHA3/Monero to have a 40/60 split:
+//   > sha3_target_time = monero_target_time * (100 - 40) / 40
+//   > monero_target_time = sha3_target_time * (100 - 60) / 60
+//   > target_time = monero_target_time * sha3_target_time / (monero_target_time + sha3_target_time)
+// `CheckDifficultyRatio` and `CheckLwmaRatios` are optional for internal testing (Network::LocalNet and Network::Igor).
+fn assert_hybrid_pow_constants(
```

Comment:
These asserts are not ideall in prodection code. 
I would say say move this to a unit test, that tests the constants of each network. 
We dont have to run assert and re check them every time on startup. 
A unit test will check them every time they are changed and checked into prodection.